subsys/bluetooth/audio/Kconfig.mediacontrol

```
@@ -0,0 +1,50 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+config BT_MEDIACONTROL
+       bool "Support for media player and media player control"
```

Given that control is enabled in other options, suggest to remove this

```suggestion
       bool "Support for media player"
```